marklogic-data-hub-central/ui/src/components/raw-data/raw-data-card-view/raw-data-card-view.tsx

```diff
@@ -67,6 +65,44 @@ const RawDataCardView = (props) => {
     }
   }
 
+  const displayDocumentSources = (item) => {
+    let sources = item.hubMetadata?.sources.map((el) => {
+      return el.name;
+    }).join(", ");
+    return sources;
+  }
+
+  const emptyField = (
+    <p className={styles.textDisabled}>none</p>
+  )
+
+  const displayDocumentMetadata = (item) => {
```

Comment:
Can you change this to "RecordMetadata"? We are trying to use the term "Record" instead of document, as Record is more familiar to a user. Additionally, "document metadata" is a specific thing in MarkLogic - https://docs.marklogic.com/xdmp.documentGetMetadata - and that differs from what we are defining as "metadata" here.